framework/rest/Serializer.php

```diff
@@ -89,6 +89,19 @@ class Serializer extends Component
      * The pagination information as shown in `_links` and `_meta` can be accessed from the response HTTP headers.
      */
     public $collectionEnvelope;
+
+    /**
+     * @var string the name of the envelope (e.g. `_links`) for returning the links objects.
+     * Works only if `collectionEnvelope` is set.
+     */
+    public $linksEnvelope = '_links';
+
+    /**
+     * @var string the name of the envelope (e.g. `_meta`) for returning the pagination object.
+     * Works only if `collectionEnvelope` is set.
+     */
+    public $metaEnvelope = '_meta';
```

Comment:
need an `@since 2.0.4` in both properties. Also please add a line in the CHANGELOG file.